prost-derive/src/lib.rs

```diff
@@ -73,14 +73,19 @@ fn try_message(input: TokenStream) -> Result<TokenStream, Error> {
         })
         .collect::<Result<Vec<_>, _>>()?;
 
+    let has_unknown_fields_accessor = fields.iter().any(|(_, f)| match f {
+        Field::UnknownFields(_) => true,
+        _ => false,
+    });
+
     // We want Debug to be in declaration order
     let unsorted_fields = fields.clone();
 
     // Sort the fields by tag number so that fields will be encoded in tag order.
     // TODO: This encodes oneof fields in the position of their lowest tag,
     // regardless of the currently occupied variant, is that consequential?
     // See: https://developers.google.com/protocol-buffers/docs/encoding#order
-    fields.sort_by_key(|&(_, ref field)| field.tags().into_iter().min().unwrap());
+    fields.sort_by_key(|&(_, ref field)| field.tags().into_iter().min().unwrap_or(std::u32::MAX));
```

Comment:
So unknown fields are always sorted last, why is that decision made? I would like that to be more explicit in the code.